pkg/catalogv2/content/content.go

```diff
@@ -95,11 +114,18 @@ func (c *Manager) Index(namespace, name string) (*repo.IndexFile, error) {
 	if err := json.Unmarshal(data, index); err != nil {
 		return nil, err
 	}
+
+	c.lock.Lock()
+	c.IndexCache[fmt.Sprintf("%s/%s", r.status.IndexConfigMapNamespace, r.status.IndexConfigMapName)] = indexCache{
+		index:      *index,
+		binaryData: cm.BinaryData["content"],
+	}
+	c.lock.Unlock()
+
 	return c.filterReleases(index), nil
 }
 
 func (c *Manager) filterReleases(index *repo.IndexFile) *repo.IndexFile {
-	index.SortEntries()
```

Comment:
Was this redundant? It's really important the data is sorted.

Reply:
https://github.com/rancher/rancher/blob/417bdfd9874f8903c5d9923f78859fb1e313a098/pkg/controllers/dashboard/helm/repo.go#L180 you did sorting at here